addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightV6SessionManager.java

```diff
@@ -239,13 +246,26 @@ public void dispose() {
      * as if the IP of a bridge has changed and the session got invalid because of that.
      *
      * A response will assign us session bytes.
+     *
+     * @throws InterruptedException
      */
-    private void send_search_for_broadcast() {
+    private void send_search_for_broadcast() throws InterruptedException {
         byte[] buf = new byte[1000];
 
         DatagramPacket p = new DatagramPacket(buf, buf.length);
         p.setPort(sendQueue.getPort());
         p.setData(search_for_packet());
+        if (lastKnownIP != null) {
+            p.setAddress(lastKnownIP);
+            try {
+                sendQueue.datagramSocket.send(p);
+                Thread.sleep(10);
+                sendQueue.datagramSocket.send(p);
+            } catch (IOException e) {
+                logger.error("Could not send discover packet! {}", e.getLocalizedMessage());
```

Comment:
1) Make this a `warn`, IOException can just happen since networks are not 100% reliable this should not be reported as an `error`.
2) Is 'discover' intentionally or a typo?